init.lua

```diff
@@ -44,6 +44,13 @@ local sharedtags = {
 local function salvage(tag)
     -- The screen to move the orphaned tag to.
     local newscreen = capi.screen.primary
+    -- the primary screen may be the one that is being
+    -- removed, so try to find a different screen if possible.
+    for s in capi.screen do
+        if s ~= current_screen then
```

Comment:
What's current_screen?

Reply:
sorry! bad copy-paste missed a variable.

`current_screen` is `tag.screen` on my local fork, I'll update this PR.